File: week-2/trapezoid.c
#include <stdio.h>

void secret_shape(int n)
{
   for(int i = 1; i <= n; i++)
   {
     for(int j = 1; j <= (i+(n>1)); j++)
     {
       printf("*");
     }
     printf("\n");
   }
   printf("\n");
}

int main()
{
   int start = 1;
   int end = 5;
   for(int i = 1; i <= 5; i++)
   {
      secret_shape(i);
   }
   return 0;
}

File: week-2/trapezoid.py
def secret_shape(n):
    for i in range(1,n+1):
        end = i+1 if n > 1 else i
        for j in range(1,end+1):
            print("*",end="")
        print()
# code for testing
for i in range(1,6):
    secret_shape(i)
    print()
